### PR TITLE
Sea changes (Limpet, ship balance)

### PR DESCRIPTION
**Pull request by Aquanim**
See list below

### CODE REVIEW

**Comment by Aquanim** [comment]
Somebody else should at least sanity check Limpet (balance-wise and bug-wise) before stable.

**Comment by Aquanim** [comment]
Summary of changes:

**New unit: Limpet (Amphibious Bomb)** 
Cost 300
HP 300
Speed 4.1
Damage 150 normal 1500 slow
AoE 500

**Reef:**
Cost 3500 -> 3000
Tacnuke replaced with disarm missile:
Cost 600 -> 150
Stockpile time 60 -> 30
12000 disarm damage, disarms for 10 seconds
280 AoE

**Ronin:**
Refire rate 4 -> 5 (Effective DPS 150 -> 120)
Weapon velocity 400 -> 350
Turret traverse rate 150 -> 50
HP 3000 -> 2500

**Siren:**
Range 380 -> 340

**Shogun:**
Cost 3800 -> 4000

Claymore changes reverted pending a more sophisticated solution
~~**Claymore:**~~
~~HP 1350 -> 1650~~
~~Reload 8 -> 6 (Effective DPS 112.5 -> 150)~~
~~AoE 290 -> 200~~
~~Projectile changes:~~
~~Starting speed 5 -> 20~~
~~Acceleration 15 -> 20~~
~~Max speed 280 -> 160~~

File: LuaUI/Configs/lang/units.en.json [hunk 1]
```diff
@@ -986,8 +986,8 @@
 	},
 	"shipcarrier" : {
 		"name" : "Reef",
-		"description" : "Aircraft Carrier (Bombardment), Stockpiles tacnukes at 10 m/s",
-		"helptext" : "The most versatile ship on the high seas, the carrier serves several functions. It is equipped with a manual-fire tactical missile launcher for long range bombardment and serves as a mobile repair base for friendly aircraft. Perhaps most notably, the carrier provides its own complement of surface attack drones to engage targets."
+		"description" : "Aircraft Carrier (Bombardment), stockpiles disarm missiles at 5 m/s",
+		"helptext" : "The most versatile ship on the high seas, the carrier serves several functions. It is equipped with a manual-fire disarming missile launcher for disabling enemy capityal ships at range and serves as a mobile repair base for friendly aircraft. Perhaps most notably, the carrier provides its own complement of surface attack drones to engage targets."
```
Comment (sprunk) [inline LuaUI/Configs/lang/units.en.json]: "capityal"

File: units/amphbomb.lua [hunk 2]
```diff
@@ -1,26 +1,27 @@
...
   collisionVolumeScales  = [[22 20 22]],
   collisionVolumeType    = [[ellipsoid]],
+  corpse                 = [[DEAD]],
 
   customParams           = {
     amph_regen = 10,
-    amph_submerged_at = 40,
- 	maxwatertank = [[100]],
+    amph_submerged_at = 30,
+    helptext       = [[The Limpet can dodge most slow projectiles thanks to its agility and small size, allowing it to get close to enemy units in order to detonate, slowing and damaging them.]],
```
Comment (sprunk) [inline units/amphbomb.lua]: Duplicate helptext, there is already one in `LuaUI/Configs/lang/units.en.json`.

File: units/amphbomb.lua [hunk 3]
```diff
@@ -1,26 +1,27 @@
 unitDef = {
   unitname               = [[amphbomb]],
```
Comment (sprunk) [inline units/amphbomb.lua]: Consider the ability to float. If the bomb detonates underwater it harms amphs more than hovers and ships.

File: LuaUI/Configs/lang/units.en.json (hunk 1 above)
Comment (Aquanim) [inline LuaUI/Configs/lang/units.en.json]: oops

File: units/amphbomb.lua (hunk 3 above)
Comment (Aquanim) [inline units/amphbomb.lua]: I think I want to see the current version first but that is a version of the unit worth contemplating.

**Pull request merged**